### ISSUE TITLE
"Could not load python37.dll" Exception

### ISSUE THREAD

**Issue by CharalamposGeorgiadis** [Unity-Technologies/com.unity.perception#430]
Hello,

I recently updated my perception project to version 0.9.0-preview.2. Ever since then, I get the same error all the time:

```
Win32Exception: The specified module could not be found.

Rethrow as DllNotFoundException: Could not load Library/PythonInstall/python37.dll
Python.Runtime.Platform.WindowsLoader.Load (System.String dllToLoad, System.String directory) (at <5dde01e97c6f480082d83366ee8e86a6>:0)
Python.Runtime.PythonEngine.InitializeLibrary (System.String library, System.String directory) (at <5dde01e97c6f480082d83366ee8e86a6>:0)
UnityEditor.Scripting.Python.PythonRunner.InstallAndLoadPython () (at Library/PackageCache/com.unity.scripting.python@4.0.0-exp.5/Editor/PythonRunner.cs:421)
UnityEditor.Scripting.Python.PythonRunner.DoEnsureInitialized () (at Library/PackageCache/com.unity.scripting.python@4.0.0-exp.5/Editor/PythonRunner.cs:681)
UnityEditor.Scripting.Python.PythonRunner.EnsureInitialized () (at Library/PackageCache/com.unity.scripting.python@4.0.0-exp.5/Editor/PythonRunner.cs:325)
UnityEditor.EditorApplication.Internal_CallDelayFunctions () (at <63ccfe108ffa44728b417cb47fb2f841>:0)
```
It does not affect the execution of the project at all but it is very annoying. I am not 100% certain that this error is connected to perception, but since it appeared after updating my perception package version I thought I would ask here if anyone knows how to solve this. The python37.dll file does exist on that directory, so it is not an issue of a missing file.

**Comment by aryan-mann** [comment]
Hi @XarisGeorgiadis!
The new visualizer functionality in the Perception package uses the `com.unity.scripting.python@4.0.0-exp.5` under the hood. From the exception, it seems like the issue is connected to this dependency. I will make a note to relay this bug to them!

From the exception, I'm assuming this is on Windows? 

Best,
Aryan

**Comment by XarisGeorgiadis** [comment]
> Hi @XarisGeorgiadis! The new visualizer functionality in the Perception package uses the `com.unity.scripting.python@4.0.0-exp.5` under the hood. From the exception, it seems like the issue is connected to this dependency. I will make a note to relay this bug to them!
> 
> From the exception, I'm assuming this is on Windows?
> 
> Best, Aryan

@aryan-mann Yes I am on Windows 10

**Comment by aryan-mann** [comment]
Hey @XarisGeorgiadis!
Could you confirm if this issue still occurs in Perception 0.10.0?

**Comment by XarisGeorgiadis** [comment]
Hello @aryan-mann

Yes, I still have get that error.

**Comment by aryan-mann** [comment]
Thanks for confirming! Does the package manager show a package called `Python for Unity`?
<img width="380" alt="image" src="https://user-images.githubusercontent.com/18741614/161647795-5e624c61-998b-49b1-a63c-866715b05c33.png">

**Comment by XarisGeorgiadis** [comment]
@aryan-mann Yes, I have version 4.0.0-exp.5, but there is an update option for version 4.0.0-pre.1

**Comment by StevenBorkman** [comment]
Hey all! The Perception team just released 🎉 Perception 1.0 🎉, a major update to the toolset! With this release we have introduced a new output format: SOLO, which is a more optimized, efficient data format. Along with this we have release python tools (pysolotools) and a new visualizer that uses Voxel51's viewer (pysolotools-fiftyone). Our old python dataset viewer has been deprecated.

There are a bunch of cool features to check out such as ray tracing, depth, a new output format SOLO, normals, and more! Here is the full list of changes: [Perception 1.0 Changelog](https://github.com/Unity-Technologies/com.unity.perception/blob/main/com.unity.perception/CHANGELOG.md)
